runtime/isp_run_app.py

```diff
@@ -106,6 +109,10 @@ def main():
         logger.error("Invalid choice of runtime. Valid choices: frtos, sel4, bare, stock_frtos, stock_sel4, stock_bare")
         return
 
+    if args.arch not in ["rv32", "rv64"]:
```

Comment:
If we want to explicitly check the arch option, we should factor this list out into `isp_utils`

Reply:
There are a bunch of other hard-coded list checks in this function.  There are also many hard-coded strings in the help output.

This can probably be done as part of a larger clean-up of all these items.